src/Uno.UI/ts/WindowManager.ts

```diff
@@ -1417,6 +1417,11 @@ namespace Uno.UI {
 				}
 			}
 
+			// This is necessary because in Safari 17 "white-space" is not selected by index (i.e. elementStyle[i])
+			// This is important to implement the Wrap/NoWrap of Controls
+			if (elementStyle.cssText.includes("white-space") && !updatedStyleString.includes("white-space"))
+				updatedStyleString += "white-space: " + elementStyle.whiteSpace + "; ";
+
```

Comment:
This change is causing an issue where visually, on chrome, the text is not wrapping anymore. In the playground, this is the expected result: https://playground.platform.uno/#e3a197e1, and if with the debugger, you take the `p` element and set it `white-space: pre` instead of `white-space: pre-wrap`, you'll notice the incorrect behavior.

Reply:
Where can I see the source code of https://playground.platform.uno/#e3a197e1 ?

Reply:
@jeromelaban 
I tested it but I didn't found what was wrong.
My behavior was as follows.

1. When TextWrapping.Wrap

> - On Windows
![image](https://github.com/unoplatform/uno/assets/122166904/b00c8a72-6b95-4611-b564-72a37a020d8d)

> - On Chrome ( with change)
![image](https://github.com/unoplatform/uno/assets/122166904/36a3e3fc-6fee-4df2-95b5-36bfc8f33f68)
`white-space: pre-wrap;`

> - On Chrome ( without change)
![image](https://github.com/unoplatform/uno/assets/122166904/4e56c48c-2ab3-4d20-aa21-e26863b87c1e)
`white-space: pre-wrap;`

2. When TextWrapping.NoWrap
> - On Windows
![image](https://github.com/unoplatform/uno/assets/122166904/34565fab-b47b-44cf-9e73-68291e0bb959)

> - On Chrome ( with change)
![image](https://github.com/unoplatform/uno/assets/122166904/4ed0a04e-b7d6-4a4a-8ed6-ff6100e40bdf)
`white-space: pre;`

> - On Chrome ( without change)
![image](https://github.com/unoplatform/uno/assets/122166904/38890290-3ec8-45b6-925c-05d10a78d6e0)
`white-space: pre;`

3. In https://playground.platform.uno/#e3a197e1 (TextWrapping="Wrap")
![image](https://github.com/unoplatform/uno/assets/122166904/c3bcf943-d181-44a5-9b0f-6ffbb3b53e6d)
`white-space: pre-wrap;`
When setting it `white-space: pre` instead `white-space: pre-wrap;` with debugger
![image](https://github.com/unoplatform/uno/assets/122166904/950567bf-8d89-458d-9649-b8e6e673b3fc)

Reply:
Testing in the playground is one way, but you can also see the visual result of the runtime test that you created for this PR. It should be showing text on multiple lines but does not (the textblock stays visible on the right side of the window, which you can check for issues).

Reply:
I think it's difficult to see the visual result because there is no delay before [changing `TextWrapping` to `NoWrap`.](https://github.com/unoplatform/uno/pull/14016/files/142d6b94308e9daa7aeeb9ea150fd11ca6328568#diff-97847806b6fa108826d10c0c9912de426d6a64d29281cc99b8dc72342580aaa0)